src/applications/Job.java

```diff
@@ -39,8 +39,8 @@ public int getJobNumber() {
     }
 
     // other methods
-    public void addTask(int theMachine, int theTime) {
-        getTaskQ().put(new Task(theMachine, theTime));
+    public void addTask(int machineIndex, int theTime) {
+        getTaskQ().put(new Task(machineIndex, theTime));
```

Comment:
I like this change. Saying explicitly that what we're using is an index makes more sense when reading this.